Set default Content-Disposition of plain text media to 'inline'

silkeh opened the pull request:
Set the default `Content-Disposition` of files with the `text/plain` content types to `inline`. This ensures that web browsers show these messages, instead of downloading them. 

This is mainly useful for the IRC bridge, as it sends long Matrix messages as links to `text/plain` files on IRC. These are now a single click away for IRC users.

turt2live reviewed (changes requested):
This steps from a large ongoing discussion around whether or not the media repo should be serving these sorts of things. I'd be okay with making this work for `text/plain`, but not for `text/*` given that opens up HTML and other things.

silkeh commented:
@turt2live fair, `text/plain` is the only MIME-type that is really needed for the Matrix bridge anyway. Maybe it would make sense to make the inline MIME-types configurable somehow?

I have updated the PR with only `text/plain` and rebased on master.

turt2live reviewed (approved):
Thanks!
